fn/templates/fnlb-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}-fnlb-depl
+  labels:
+    app: {{ template "fullname" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+       labels:
+         app: {{ template "fullname" . }}
+         chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+         heritage: {{ .Release.Service }}
+         release: {{ .Release.Name }}
+         role: fn-lb
+    spec:
+      containers:
+        - name: fnlb
+          image: {{ .Values.fnlb.image }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          resources:
+{{ toYaml .Values.fnlb.resources | indent 12 }}
+          ports:
+            - containerPort: 8081
+            - containerPort: 8082
+          readinessProbe:
+            httpGet:
+              path: /1/lb/dash
```

Comment:
this one may be triggering metrics collection for the currently-super-hokey debugging dashboard. anyway this is probably best hidden for now (shame on me) -- we don't have a simple ping / version endpoint on lb, I added to the mini lb TODO card https://github.com/fnproject/fn/issues/115